api/dataset_test.go

```diff
@@ -85,7 +85,7 @@ func TestDatasetHandlers(t *testing.T) {
 		{"GET", "/export/me/cities/at/map/QmPRjfgUFrH1GxBqujJ3sEvwV3gzHdux1j4g8SLyjbhwot", nil},
 		{"DELETE", "/", nil},
 	}
-	runHandlerTestCases(t, "export", h.ZipDatasetHandler, exportCases, true)
+	runHandlerTestCases(t, "export", h.ZipDatasetHandler("/export"), exportCases, true)
```

Comment:
This is fine for now, but fyi, we are working on removing `/export`, and will soon require callers to switch to get. See https://github.com/qri-io/qri/issues/1176. Hoping to remove export after 0.9.9 is released.

Reply:
Yep, once we start the move away from /export, plan is a) we continue to support the usecase b) happy to update to the new codepath, just that it's a bit too much work to unblock download just now.